server/cluster/cluster.go

```diff
@@ -44,6 +50,31 @@ func (c *Cluster) Name() string {
 	return c.metaData.cluster.Name
 }
 
+func (c *Cluster) init(ctx context.Context, shards []*clusterpb.Shard) error {
```

Comment:
It is confusing when both `init` and `Load` methods exists at the same time, and some comments are necessary to explain how to call these methods.